examples/gcp-with-psc-exfiltration-protection/README.md

```diff
@@ -0,0 +1,84 @@
+# Provisioning Databricks on GCP workspace with a Hub & Spoke network architecture for data exfiltration protection
+
+This example is using the [gcp-with-psc-exfiltration-protection](../../modules/gcp-with-psc-exfiltration-protection) module.
```

Comment:
We need to put a warning at beginning that PSC isn't enabled by default and user should contact databricks team.